src/route/index.js

```diff
@@ -2,48 +2,27 @@ import { createWebHistory, createRouter } from 'vue-router';
 
 import AppLogin from '../views/AppLogin.vue';
 import AppError from '../views/AppError.vue';
-import Dashboard from '../views/dashboard/Dashboard.vue';
+import DashboardView from '../views/dashboard/Dashboard.vue';
```

Comment:
Same comment as the other day, your component have to be multi-word : https://v3.vuejs.org/style-guide/#multi-word-component-names-essential

To follow our own guide, it has to be DashboardPage : https://github.com/heypongo/PongoDashboardVue#structure

So your file have to be renamed to `../views/dashboard/DashboardPage.vue`